src/lib/seam/components/DeviceDetails/ThermostatDeviceDetails.tsx

```diff
@@ -234,6 +238,206 @@ function FanModeRow({ device }: { device: ThermostatDevice }): JSX.Element {
   )
 }
 
+function ClimateSettingRow({
+  device,
+}: {
+  device: ThermostatDevice
+}): JSX.Element {
+  const deviceHeatValue =
+    device.properties.current_climate_setting.heating_set_point_fahrenheit
```

Comment:
At a later point, we need to add the temperature unit to some state context to feed them in to lower components. This is far beyond the scope of this PR, however.